Src/PerformanceCollector/Filtering/AccumulatedValue.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System.Collections.Concurrent;
+
+    /// <summary>
+    /// Accumulator for operationalized metrics.
+    /// </summary>
+    internal class AccumulatedValue
+    {
+        public readonly ConcurrentStack<double> Value = new ConcurrentStack<double>();
```

Comment:
This particular accumulator works by accumulating values as they pass through the pipeline in a list. The aggregation happens once a second elsewhere, this is just a DTO. The word "to accumulate" is used in its original meaning, which is "to gather together".